bootstrap-page.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<head>
+    <title>Bootstrap Page</title>
+    <style>
+        .container {
+            float: left;
+            width: 50%;
+        }
+        .container-2 {
+            float: right;
+            width:50%;
+        }
+        .carbon-ad {
+            float:left;
+            width: 100%;
+            border: 3px;
+        }
+        .container-3 {
+            float: none;
+            width: 50%;
+        }
+        .footer {
+            width: 25%;
+        }
+    </style>
+</head>
+<body>
+    <div>
+        <nav>
+            <ul style="list-style:none;">
```

Comment:
You usually do not in-line style like this.
Instead, attach a class to this `ul`, and then add the corresponding css to that selector